python/graphstorm/inference/lp_infer.py

```diff
@@ -43,6 +43,7 @@ class GSgnnLinkPredictionInfer(GSInfer):
     # TODO(zhengda) We only support full-graph inference for now.
     def infer(self, data, loader, save_embed_path,
             edge_mask_for_gnn_embeddings='train_mask',
+            use_mini_batch_infer=False,
```

Comment:
should we have this as an argument? if mini-batch inference is used for custom models, we can easily test if the input model is a custom model.